docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,59 @@
+## Checklist
+<!--
+Mark an `[x]` for completed items.
+-->
+
+* [ ] Update CHANGELOG.rst  for each commit separately.
```

Comment:
- Pay attention to always write entries under the "Unreleased" section
- Mark all breaking changes as `**BACKWARD INCOMPATIBLE:**` and put them before non-breaking changes